.github/workflows/push-images.yml

```diff
@@ -69,7 +71,9 @@ jobs:
 
       - name: Push Model Server Image
         run: |
-          for tag in ${{ steps.modelmeta.outputs.tags }}; do
+          set -x
+
+          for tag in ${{ join(steps.modelmeta.outputs.tags, ' ') }}; do
```

Comment:
`${{ steps.modelmeta.outputs.tags }}` results in a multi-line string, hence join:

https://github.com/mlflow/mlflow/actions/runs/4412341741/jobs/7731711749

```
  for tag in ghcr.io/mlflow/model-server:v2.2.2
  ghcr.io/mlflow/model-server:latest; do
    docker tag model-server:latest $tag
    docker push $tag
  done
```